photon-core/src/main/java/org/photonvision/vision/camera/LibcameraGpuSettables.java

```diff
@@ -164,8 +161,7 @@ public void setBrightness(int brightness) {
     @Override
     public void setGain(int gain) {
         lastGain = gain;
-        // TODO units here seem odd -- 5ish seems legit? So divide by 10
-        var success = LibCameraJNI.setAnalogGain(r_ptr, gain / 10.0);
+        var success = LibCameraJNI.setAnalogGain(r_ptr, MathUtils.map(gain, 0.0, 100.0, 1, 10));
```

Comment:
Why 1 to 10 instead of 0-10?

Reply:
Gahhh this stuff shouldn't be in this PR!